app/src/common/shared/org/mozilla/vrbrowser/geolocation/GeolocationWrapper.kt

```diff
@@ -0,0 +1,40 @@
+package org.mozilla.vrbrowser.geolocation
+
+import android.content.Context
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.future.future
+import kotlinx.coroutines.launch
+import mozilla.components.service.location.MozillaLocationService
+import org.mozilla.vrbrowser.browser.engine.EngineProvider
+import org.mozilla.vrbrowser.browser.SettingsStore
+import java.util.concurrent.CompletableFuture
+
+object GeolocationWrapper {
+
+    fun update(context: Context) {
+        val locationService = MozillaLocationService(
+                context,
+                EngineProvider.getDefaultClient(context),
+                org.mozilla.vrbrowser.BuildConfig.MLS_TOKEN
```

Comment:
What happens if the MLS_TOKEN is an empty string (`""`)?

Reply:
We default to the `LocaleSearchLocalizationProvider` as a search engine provider so it would use whatever the system locale is.